### PR TITLE
POC: MINIFICPP-738 - EL should be able to access global properties

### PR DESCRIPTION
**Pull request by arpadboda**
Thank you for submitting a contribution to Apache NiFi - MiNiFi C++.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced
     in the commit message?

- [ ] Does your PR title start with MINIFICPP-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [ ] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE file?
- [ ] If applicable, have you updated the NOTICE file?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: libminifi/test/expression-language-tests/ExpressionLanguageTests.cpp [hunk 1]
```diff
@@ -203,6 +209,12 @@ TEST_CASE("GetFile PutFile dynamic attribute", "[expressionLanguageTestGetFilePu
       get_file,
       processors::GetFile::KeepSourceFile.getName(),
       "false");
+  auto update = plan->addProcessor(
+      "UpdateAttribute",
+      "UpdateAttribute",
+      core::Relationship("success", "description"),
+      true);
+  update->setDynamicProperty("prop_attr", "${'nifi.my.own.property'}_added");
```
Comment (arpadboda) [inline libminifi/test/expression-language-tests/ExpressionLanguageTests.cpp]: @jdye64 is this what you need?

Comment (jdye64) [reply libminifi/test/expression-language-tests/ExpressionLanguageTests.cpp]: Yes it is. However I just want to make sure we document that those global properties must start with "nifi." could we add that to the documentation somewhere?

Comment (arpadboda) [reply libminifi/test/expression-language-tests/ExpressionLanguageTests.cpp]: Sure, that's planned. Just a POC yet to make sure it's what we need. 
Maybe @phrocker has some better idea to access global configuration here.

**Comment by phrocker** [comment]
@apiri #489 actually contains this commit which I'm +1. Would you prefer I rebase that pr after a merge of this or can we just take care of it all at merge time?

**Comment by apiri** [comment]
@phrocker Yep, that works for me.  I'll be sure to close this out when I merge the other.

**Comment by apiri** [comment]
Closed via f01953b8cd8814fa5b4faa0a7cb588f0545bb231

**Pull request closed without merging**